Source/WebInspectorUI/UserInterface/Models/FontStyles.js

```diff
@@ -312,6 +316,9 @@ WI.FontStyles = class FontStyles
             }
         }
 
+        // returns true if the CSS property was authored with `!important` in its value (which exists on WI.CSSProperty._rawValue)
```

Comment:
The code comment is redundant. Please remove it. I had put it in the sniped in my review as an explanation.

Reply:
Removed